docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,234 @@
+and the AdoptOpenJDK Java compiler and runtime.
 
-**Coming soon.**
+Create an Android app and compile it
+====================================
+
+Firs, run the ``create`` command. This downloads an Android app template and
+adds your Python code to it.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      (beeware-venv) C:\...>briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+Once this completes, we'll now have an ``android`` directory in your project.
+This directory will contain a ``Hello World`` folder, which will contain an
+Android project with a Gradle build configuration. This project will contain
+your application code, and a support package containing the Python interpreter
+and the `rubicon-java` library.
+
+In the process, Briefcase downloads the Android SDK. You may be prompted to
+accept Google's Android SDK license.
+
+You can then use Briefcase's ``build`` command to compile this into an Android
+APK app file. It will take quite a few minutes the first time, depending on
+your Internet connection speed, and then should take less than 30 seconds every
+successive time.
+
+.. admonition:: Gradle can look stuck
+
+  You may see Gradle stuck on the "CONFIGURING" step for many, many minutes.
+  Gradle is likely busy performing downloads, and you should not interrupt it.
+  This typically only affects the first time an app is built.
+
+.. code-block:: bash
```

Comment:
Should be put into a tab to provide Linux/macOS/Windows specific appearnace.

Reply:
Fixed in 0994883